frontend/src/App/App.tsx

```diff
@@ -83,6 +87,12 @@ function App() {
     setMurals(data.rows);
   };
 
+  useEffect(() => {
```

Comment:
I think as files get larger, we should start adding a basic docstring over functions, especially useEffects (since they don't self document through their naming) to make it easier to parse which function does what

Reply:
good call, I'll add some